compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/validator/ServiceAnalysisTask.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
This is actually a file rename. GitHub incorrectly identified it as a file deletion + new file addition.